packager/VisibilityChecker.cc

```diff
@@ -487,11 +486,15 @@ class VisibilityCheckerPass final {
                         }
                     }
                 } else if (importedAsTest) {
+                    ENFORCE(!isTestImport);
                     if (auto e = ctx.beginError(lit.loc(), core::errors::Packager::UsedTestOnlyName)) {
                         e.setHeader("Used `{}` constant `{}` in non-test file", "test_import", litSymbol.show(ctx));
                         auto &pkg = ctx.state.packageDB().getPackageInfo(otherPackage);
                         if (auto exp = this->package.addImport(ctx, pkg, false)) {
                             e.addAutocorrect(std::move(exp.value()));
+                            if (!db.errorHint().empty()) {
+                                e.addErrorNote("{}", db.errorHint());
+                            }
```

Comment:
We typically put the `addErrorNote` last, would you mind moving it for consistency?

Reply:
Ok, moved the `addErrorLine` call up.